lib/model/store.dart

```diff
@@ -413,6 +416,11 @@ class PerAccountStore extends ChangeNotifier with EmojiStore, ChannelStore, Mess
   /// For docs, please see [InitialSnapshot.realmWaitingPeriodThreshold].
   final int realmWaitingPeriodThreshold;  // TODO(#668): update this realm setting
   final int maxFileUploadSizeMib; // No event for this.
+  final String? _realmEmptyTopicDisplayName; // TODO(#668): update this realm setting
+  String get realmEmptyTopicDisplayName {
+    assert(_realmEmptyTopicDisplayName != null); // TODO(log)
+    return _realmEmptyTopicDisplayName ?? 'general chat';
+  }
```

Comment:
nit: getter first: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#order-other-class-members-in-a-way-that-makes-sense